src/components/Login/Login.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router';
+import { Button, Form, Spinner } from 'react-bootstrap';
+import { useCookies } from 'react-cookie';
+import axios from 'axios';
+
+import {
+    loginPending,
+    loginSuccessful,
+    loginFail,
+} from '../../redux/authSlice';
+import { setMessage } from '../../redux/messageAlertSlice';
+
+let Login = () => {
+    const [userName, setUserName] = useState('');
+    const [userPassword, setUserPassword] = useState('');
+    const [rememberMe, setRememberMe] = useState(false);
+    const [_, setCookie] = useCookies(['authToken']);
```

Comment:
why we are using underscore at first index?

Reply:
basically this library doesnt allow importing only setCookie,
it allows the follow syntax
const [cookie, setCookie] = useCookie(['authToken']
cost [cookie, setCookie,removeCookie] = useCookie['authToken']
as i was not goona use cookie anywhere so i just replaced it with _ . like in python it means whatever is declared here is not important is not used.

Reply:
but i have replaced it with cookie